tests/test_sklearn.py

```diff
@@ -6,6 +6,6 @@
 
 from sklearn.utils.estimator_checks import check_estimator
 
-check_estimator(graspologic.embed.AdjacencySpectralEmbed)
+# check_estimator(graspologic.embed.AdjacencySpectralEmbed)  # not a valid sklearn estimator with oos predict method in current implementation
```

Comment:
This breaks in the debugger when sklearn expects to allow a y-value in `ase.predict` with shape `(10, 3)` (with `self.latent_left_` also having shape `(10, 3)`, fit on a 10x10 adjacency matrix).

This array shape would be allowed in a normal `predict` method, but doesn't make sense for out-of-sample prediction.

Reply:
not sure if worth trying to maintain compatibility here, could change the name of function or just remove. any thoughts @j1c?

Reply:
I think its fine to remove.

Reply:
9be3250